src/noshow/dashboard/connection.py

```diff
@@ -61,20 +61,22 @@ def get_patient_list(_session: Session, date_input: date) -> List[str]:
     threshold_date = date.today() - timedelta(days=round(MUTE_PERIOD * 30.5))
 
     call_list = _session.execute(
-        select(ApiPrediction.patient_id, func.max(ApiPrediction.prediction))
+        select(
+            ApiPrediction.patient_id,
+            ApiPatient.treatment_group,
```

Comment:
the query is auto-optimized right? or is this changed order faster?

Reply:
I think it's auto-optimized, but it makes more sense for me to order them like a query. And I had to add a little bit for the treatment group anyways